packages/components/document-card/README.md

```diff
@@ -0,0 +1,131 @@
+<h1 align="center">
+    <a href="https://tradeshift.com/">
+      <img alt="Tradeshift" src="https://tradeshift.com/wp-content/themes/Tradeshift/img/brand/logo-black.png"/>
+    </a>
+</h1>
+
+<h1 align="center">Elements - document-card</h1>
+
+<p align="center">
+  Part of the reusable Tradeshift UI Components as Web Components.
+    <a href="https://tradeshift.github.io/elements/?path=/story/ts-document-card--default">
+      Demo
+    </a>
+</p>
+
+<p align="center">
+    <a href="https://www.npmjs.com/package/@tradeshift/elements.document-card">
+      <img alt="NPM Version" src="https://badgen.net/npm/v/@tradeshift/elements.document-card" height="20"/>
+    </a>
+    <a href="https://npmcharts.com/compare/@tradeshift/elements.document-card?minimal=true">
+		  <img alt="Downloads per month" src="https://badgen.net/npm/dm/@tradeshift/elements.document-card" height="20"/>
+		</a>
+		<a href="https://www.npmjs.com/browse/depended/@tradeshift/elements.document-card">
+		  <img alt="Dependent packages" src="https://badgen.net/npm/dependents/@tradeshift/elements.document-card" height="20"/>
+		</a>
+</p>
+
```

Comment:
For new components we also documenting the available `Properties`, `Events` and `Slots`. You can use the `modal` component as an example.
To not block your development process you can leave a TODO to add this information later.

Same for `status` and `table` components.